docs/getting-started/smr-disk.md

```diff
@@ -89,7 +89,7 @@ The possible values of the *zoned* attribute are shown in the table below.
 
 </center>
 
-The kernel messages also include useful information on the disk.
+Kernel-ring-buffer messages also contain useful information about the disk:
```

Comment:
Please remove "ring-buffer". This is a kernel implementation detail that is not relevant here.

Reply:
I've removed it.